pkg/didcomm/protocol/didexchange/models.go

```diff
@@ -11,11 +11,32 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
 )
 
+// OOBInvitation to connect with did-exchange.
+type OOBInvitation struct {
+	// ID of this invitation (for record-keeping purposes).
+	// TODO can we remove this?
+	ID string
+	// TODO remove
+	Type string `json:"@type"`
```

Comment:
just curious, why only one filed has json tag and not others ?

Reply:
To reuse the existing machinery in the didexchange protocol service (`service.DIDCommMsg.Type()`). The existing code only expects didcomm messages. This new struct field is *not* a didcomm message.